### ISSUE TITLE
Char _ is allowed number separatist

### ISSUE THREAD

**Issue by Alexey-T** [digitalcora/sublime-text-gleam#6]
```
  # Numbers
  number:
    - match: \b0b[01]+\b
      scope: constant.numeric.binary.gleam
    - match: \b0o[0-7]+\b
      scope: constant.numeric.octal.gleam
    - match: \b[[:digit:]][[:digit:]_]*(\.[[:digit:]]*(e-?[[:digit:]]+)?)?\b
      scope: constant.numeric.decimal.gleam
    - match: \b0x[[:xdigit:]]+\b
      scope: constant.numeric.hexadecimal.gleam
```
'_' is valid digit separatist now.

**Comment by digitalcora** [comment]
Thank you for the catch! This is fixed in version 0.2.10, which should be published to Package Control shortly.

**Comment by Alexey-T** [comment]
Your rule dont support 12.3_567

**Comment by Alexey-T** [comment]
And 1.2e3_45

**Comment by digitalcora** [comment]
Thanks for the added test cases. The new version had not made it to Package Control yet, so I have fixed these and re-released version 0.2.10.
